src/client/NativeClient.js

```diff
@@ -50,6 +50,7 @@ export const Network = {
 
     MAINNET: {
         "https://grpc-web.myhbarwallet.com:443": new AccountId(3),
+        "https://node01-00-grpc.swirlds.com:443": new AccountId(4),
```

Comment:
We need to add the following GRPC proxies to Native and Web client as well:
https://node02.swirldslabs.com/
https://node03.swirldslabs.com/
https://node04.swirldslabs.com/
https://node05.swirldslabs.com/
https://node06.swirldslabs.com/
https://node07.swirldslabs.com/
https://node08.swirldslabs.com/
https://node09.swirldslabs.com/
https://node10.swirldslabs.com/
https://node11.swirldslabs.com/
https://node12.swirldslabs.com/
https://node13.swirldslabs.com/
https://node14.swirldslabs.com/
https://node16.swirldslabs.com/
https://node17.swirldslabs.com/
https://node18.swirldslabs.com/
https://node19.swirldslabs.com/
https://node20.swirldslabs.com/
https://node21.swirldslabs.com/
https://node22.swirldslabs.com/
https://node23.swirldslabs.com/
https://node24.swirldslabs.com/
https://node25.swirldslabs.com/
https://node26.swirldslabs.com/

The nodeAccountIds are here: https://status.hedera.com/

Reply:
Please note that node01 and node15 are MISSING. This is on purpose!

Reply:
Added